src/app/shared/formly/formly.module.ts

```diff
@@ -41,7 +41,7 @@ import { ValidationWrapperComponent } from './wrappers/validation-wrapper/valida
     CaptchaExportsModule,
     CommonModule,
     DirectivesModule,
-    FormlyBaseModule.forRoot({
+    FormlyBaseModule.forChild({
```

Comment:
This is not a problem. I double checked the formly source code and there should not be any problems with this :)